apps/web/src/components/document/document-expiry-settings.tsx

```
@@ -0,0 +1,63 @@
+type DocumentExpirySettingsProps = {
+  onChange: (value: number | undefined, unit: 'day' | 'week' | 'month' | undefined) => void;
+};
+
+export const DocumentExpirySettings = ({ onChange }: DocumentExpirySettingsProps) => {
+  const [expiryValue, setExpiryValue] = useState<number | undefined>(undefined);
+  const [expiryUnit, setExpiryUnit] = useState<'day' | 'week' | 'month'>();
+  const { _ } = useLingui();
+
+  const handleExpiryValueChange = (value: string) => {
+    const parsedValue = parseInt(value, 10);
+    if (isNaN(parsedValue)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)